src/main/kotlin/insight/ColorUtil.kt

```diff
@@ -193,7 +193,7 @@ fun UElement.setColor(color: String) {
     sourcePsi.containingFile.runWriteAction {
         val project = sourcePsi.project
         val parent = this.uastParent
-        val newColorRef = generationPlugin?.getElementFactory(project)?.createQualifiedReference(color, this)
+        val newColorRef = generationPlugin?.getElementFactory(project)?.createQualifiedReference(color, sourcePsi)
```

Comment:
Can you fix all the compile errors related to downgrading intellij again? Such as this one here.

Reply:
Yes, however I’m currently on vacation and don’t have access to my pc so it will have to wait till I have access

Reply:
Used https://gitpod.io